webui/react/src/services/decoder.ts

```diff
@@ -267,31 +272,31 @@ export const jsonToTrialDetails = (data: unknown): TrialDetails => {
 };
 
 const experimentStateMap = {
-  [Determinedexperimentv1State.UNSPECIFIED]: RunState.Unspecified,
-  [Determinedexperimentv1State.ACTIVE]: RunState.Active,
-  [Determinedexperimentv1State.PAUSED]: RunState.Paused,
-  [Determinedexperimentv1State.STOPPINGCANCELED]: RunState.StoppingCanceled,
-  [Determinedexperimentv1State.STOPPINGCOMPLETED]: RunState.StoppingCompleted,
-  [Determinedexperimentv1State.STOPPINGERROR]: RunState.StoppingError,
-  [Determinedexperimentv1State.CANCELED]: RunState.Canceled,
-  [Determinedexperimentv1State.COMPLETED]: RunState.Completed,
-  [Determinedexperimentv1State.ERROR]: RunState.Errored,
-  [Determinedexperimentv1State.DELETED]: RunState.Deleted,
-};
-
-export const decodeExperimentState = (data: Determinedexperimentv1State): RunState => {
+  [AB.Determinedexperimentv1State.UNSPECIFIED]: RunState.Unspecified,
```

Comment:
this was a sync topic. but thoughts on using this kind of import if there are more than n items to import?

Reply:
yah makes sense to do this instead of calling out every import up above